docker/Dockerfile.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import textwrap
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--docker-version', required=True)
+
+args = parser.parse_args()
+
+
+dependencies = ' '.join(('git',
+                         'python-dev',
+                         'python-pip',
+                         'wget',
+                         'curl',
+                         'apt-transport-https',
+                         'ca-certificates'))
+docker_version = args.docker_version
+print textwrap.dedent("""
+    FROM ubuntu:14.04
```

Comment:
We should look at basing this image on the Toil appliance. Can you open a ticket?